tribe/settings.py

```diff
@@ -280,8 +286,6 @@
         #'release': raven.fetch_git_sha(dirname(__file__)),
     }
 
-# GOOGLE ANALYTICS
-GOOGLE_ANALYTICS_KEY = secrets.get('ga', 'GOOGLE_ANALYTICS_KEY')
```

Comment:
You moved GA_CODE but completely removed this var. Was this intended?

Reply:
`secrets.ini` defined  `GOOGLE_ANALYTICS_KEY`, and `production.ini` defined `GA_CODE`. Both of them refer to the same value. But Django is using `GA_CODE` only (in `urls.py`). So the one defined in `secrets.ini` is redundant. That's why I moved `GA_CODE` from `production.ini` to `secrets.ini`.

Reply:
By the way, I have tested the new backend config.

Reply:
`secrets.ini` is supposed to be a file provided by deployment. So it is encrypted and included in the `tribe-ec2-deploy` repo on bitbucket (as you may remember).